subclass-error.js

```diff
@@ -27,11 +28,11 @@
       // stack "hack"
       var sample = (new Error())
       if (!sample.stack) sample.stack = ''
-      var goodStack = sample.stack.split('\n')
-      goodStack.splice(1, 1)
-      goodStack[0] = name
-      if (message) goodStack[0] += ': ' + message
-      this.stack = goodStack.join('\n')
+      var niceStack = sample.stack.split('\n')
+      niceStack.splice(1, 1)
+      niceStack[0] = name
+      if (message) niceStack[0] += ': ' + message
+      this.stack = niceStack.join('\n')
```

Comment:
Comment on some part of the code that won't change after force-pushing/squash/ammend.